singer-io/tap-mailshake
# Properly Handle API Limits

# Description of change
(write a short description or paste a link to JIRA)
* Properly back off when Rate limit hit for Mailshake API.
# Manual QA steps
 - Should be sufficient to sync the tap for a wide enough date range to force this to happen.
 - Tap should not quit out when mailshake rate limit is hit.
 
# Risks
 - None
 
# Rollback steps
 - revert this branch
